test/benchmark.hs

```diff
@@ -27,18 +35,29 @@
   * emap on PATRICIA tree: ...................
     4.532ns per iteration / 220663.09 per second.
 -}
+{-# LANGUAGE ScopedTypeVariables #-}
+
```

Comment:
Yes. Most projects I've seen seem to prefer keeping them separate, but if you prefer them joined I can do that.